kf4cic.c

```diff
@@ -211,6 +211,42 @@ static EFI_STATUS load_and_start_tos(void)
 
 	return ret;
 }
+
+static EFI_STATUS update_rollback_indexes()
+{
+	EFI_STATUS ret;
+	AvbOps *ops;
+	AvbSlotVerifyResult verify_result;
+	AvbSlotVerifyData *slot_data = NULL;
+	UINT8 boot_state = BOOT_STATE_GREEN;
+	const char *requested_partitions[] = {"tos",
+		NULL};
```

Comment:
MINOR: If you copied this from kf4abl.c, NULL could be in same line with "tos". Line 222 is not yet too long.

Reply:
resolved.